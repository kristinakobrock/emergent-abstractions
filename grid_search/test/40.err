  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 23.58it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 29.03it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 24.36it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 25.77it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 26.75it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 26.64it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 23.93it/s] 50%|##############################################                                              | 25/50 [00:00<00:00, 25.48it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 25.53it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 26.68it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 26.18it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 25.74it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 25.60it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.62it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 27.35it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 26.67it/s]100%|############################################################################################| 50/50 [00:01<00:00, 26.15it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 19.23it/s] 38%|###################################7                                                         | 5/13 [00:00<00:00, 25.16it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 22.15it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 22.85it/s]100%|############################################################################################| 13/13 [00:00<00:00, 22.85it/s]
