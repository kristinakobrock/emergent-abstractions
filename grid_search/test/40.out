# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6964589953422546, "acc": 0.5599859356880188, "length": 3.546933650970459, "mode": "train", "epoch": 1}
{"loss": 0.7253692150115967, "acc": 0.6292968392372131, "length": 3.91796875, "mode": "test", "epoch": 1}
{"loss": 0.6737595796585083, "acc": 0.6275466084480286, "length": 3.5197551250457764, "mode": "train", "epoch": 2}
{"loss": 0.6904618740081787, "acc": 0.5230469107627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6682186126708984, "acc": 0.6171782612800598, "length": 3.463353395462036, "mode": "train", "epoch": 3}
{"loss": 0.6865032911300659, "acc": 0.5474609732627869, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6620269417762756, "acc": 0.6272687911987305, "length": 3.4637370109558105, "mode": "train", "epoch": 4}
{"loss": 0.6737282872200012, "acc": 0.6234375834465027, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.661948025226593, "acc": 0.6278234124183655, "length": 3.466980457305908, "mode": "train", "epoch": 5}
{"loss": 0.6811070442199707, "acc": 0.5923828482627869, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6608983278274536, "acc": 0.6275531053543091, "length": 3.4571139812469482, "mode": "train", "epoch": 6}
{"loss": 0.6774053573608398, "acc": 0.605877697467804, "length": 1.08203125, "mode": "test", "epoch": 6}
{"loss": 0.6609479188919067, "acc": 0.627848744392395, "length": 3.494352102279663, "mode": "train", "epoch": 7}
{"loss": 0.6572277545928955, "acc": 0.6282898187637329, "length": 2.96875, "mode": "test", "epoch": 7}
{"loss": 0.6596342325210571, "acc": 0.6277418732643127, "length": 3.51599383354187, "mode": "train", "epoch": 8}
{"loss": 0.6581413149833679, "acc": 0.6302521824836731, "length": 3.9296875, "mode": "test", "epoch": 8}
{"loss": 0.659553050994873, "acc": 0.6277747750282288, "length": 3.5267765522003174, "mode": "train", "epoch": 9}
{"loss": 0.6581913232803345, "acc": 0.6300780773162842, "length": 3.98828125, "mode": "test", "epoch": 9}
{"loss": 0.6585545539855957, "acc": 0.6278131008148193, "length": 3.564937114715576, "mode": "train", "epoch": 10}
{"loss": 0.6542883515357971, "acc": 0.6294922232627869, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.6567061543464661, "acc": 0.6279201507568359, "length": 3.608158588409424, "mode": "train", "epoch": 11}
{"loss": 0.650484025478363, "acc": 0.632031261920929, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.6514049768447876, "acc": 0.6279398202896118, "length": 3.698328971862793, "mode": "train", "epoch": 12}
{"loss": 0.6478068828582764, "acc": 0.6339843273162842, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.6457103490829468, "acc": 0.6311712861061096, "length": 3.804137706756592, "mode": "train", "epoch": 13}
{"loss": 0.6420803070068359, "acc": 0.6357421278953552, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.6357502341270447, "acc": 0.6418427228927612, "length": 3.9018771648406982, "mode": "train", "epoch": 14}
{"loss": 0.6356587409973145, "acc": 0.6341796517372131, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.6250079274177551, "acc": 0.6564178466796875, "length": 3.9479527473449707, "mode": "train", "epoch": 15}
{"loss": 0.6332361698150635, "acc": 0.6406249403953552, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.6182278990745544, "acc": 0.660448431968689, "length": 3.9697885513305664, "mode": "train", "epoch": 16}
{"loss": 0.633442759513855, "acc": 0.6357421875, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.6132273077964783, "acc": 0.6633459329605103, "length": 3.9809305667877197, "mode": "train", "epoch": 17}
{"loss": 0.635150134563446, "acc": 0.6373047232627869, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.6091925501823425, "acc": 0.6746426820755005, "length": 3.988166570663452, "mode": "train", "epoch": 18}
{"loss": 0.6293103098869324, "acc": 0.6435547471046448, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.6038631796836853, "acc": 0.6757293939590454, "length": 3.9889962673187256, "mode": "train", "epoch": 19}
{"loss": 0.6327710151672363, "acc": 0.6464844346046448, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.601556122303009, "acc": 0.6817930936813354, "length": 3.9917004108428955, "mode": "train", "epoch": 20}
{"loss": 0.6282085180282593, "acc": 0.6455078125, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5974038243293762, "acc": 0.6808508634567261, "length": 3.990534543991089, "mode": "train", "epoch": 21}
{"loss": 0.621730625629425, "acc": 0.658007800579071, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5921661257743835, "acc": 0.6884705424308777, "length": 3.9918527603149414, "mode": "train", "epoch": 22}
{"loss": 0.6163803935050964, "acc": 0.6615234017372131, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.5907170176506042, "acc": 0.6924597024917603, "length": 3.9959700107574463, "mode": "train", "epoch": 23}
{"loss": 0.6097304821014404, "acc": 0.6740235090255737, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.5844663977622986, "acc": 0.6936630010604858, "length": 3.995353937149048, "mode": "train", "epoch": 24}
{"loss": 0.6023955941200256, "acc": 0.6826171278953552, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.5770535469055176, "acc": 0.6973457932472229, "length": 3.998187780380249, "mode": "train", "epoch": 25}
{"loss": 0.6010883450508118, "acc": 0.685546875, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.5728457570075989, "acc": 0.7016645669937134, "length": 3.9970712661743164, "mode": "train", "epoch": 26}
{"loss": 0.5979950428009033, "acc": 0.6820312142372131, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.5708958506584167, "acc": 0.7004075646400452, "length": 3.9975333213806152, "mode": "train", "epoch": 27}
{"loss": 0.5937137603759766, "acc": 0.6910156607627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.5636399984359741, "acc": 0.7034187316894531, "length": 3.998507499694824, "mode": "train", "epoch": 28}
{"loss": 0.5899524688720703, "acc": 0.689453125, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.562712550163269, "acc": 0.7025506496429443, "length": 3.995943546295166, "mode": "train", "epoch": 29}
{"loss": 0.5878274440765381, "acc": 0.6898437142372131, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.5614469647407532, "acc": 0.7069197297096252, "length": 3.99298357963562, "mode": "train", "epoch": 30}
{"loss": 0.5870451927185059, "acc": 0.6935547590255737, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.5578579306602478, "acc": 0.7052854299545288, "length": 3.9992687702178955, "mode": "train", "epoch": 31}
{"loss": 0.582696795463562, "acc": 0.6949219107627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.5581248998641968, "acc": 0.7065457701683044, "length": 3.998501777648926, "mode": "train", "epoch": 32}
{"loss": 0.5866382122039795, "acc": 0.6957031488418579, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.5533034205436707, "acc": 0.7101485133171082, "length": 3.9989876747131348, "mode": "train", "epoch": 33}
{"loss": 0.5809928178787231, "acc": 0.6937499642372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.551456868648529, "acc": 0.7094717025756836, "length": 3.996605396270752, "mode": "train", "epoch": 34}
{"loss": 0.5830825567245483, "acc": 0.6953125, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.5500871539115906, "acc": 0.7084090709686279, "length": 3.9993696212768555, "mode": "train", "epoch": 35}
{"loss": 0.579597532749176, "acc": 0.6964843273162842, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.5504238605499268, "acc": 0.7106322050094604, "length": 3.9990856647491455, "mode": "train", "epoch": 36}
{"loss": 0.579028308391571, "acc": 0.6943359375, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.5463188886642456, "acc": 0.7148914337158203, "length": 3.9983301162719727, "mode": "train", "epoch": 37}
{"loss": 0.5749126672744751, "acc": 0.7005859017372131, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.5453778505325317, "acc": 0.7157143950462341, "length": 3.9964098930358887, "mode": "train", "epoch": 38}
{"loss": 0.572807788848877, "acc": 0.7056640982627869, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.5404454469680786, "acc": 0.7195091247558594, "length": 3.9976017475128174, "mode": "train", "epoch": 39}
{"loss": 0.5718344449996948, "acc": 0.7037109732627869, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.5389717817306519, "acc": 0.7214454412460327, "length": 3.9992260932922363, "mode": "train", "epoch": 40}
{"loss": 0.566816508769989, "acc": 0.7095703482627869, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.5358168482780457, "acc": 0.7297950983047485, "length": 3.9934844970703125, "mode": "train", "epoch": 41}
{"loss": 0.5656557083129883, "acc": 0.7091797590255737, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.5312382578849792, "acc": 0.7341296076774597, "length": 3.996415138244629, "mode": "train", "epoch": 42}
{"loss": 0.5631052851676941, "acc": 0.7121094465255737, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.528256356716156, "acc": 0.7330861687660217, "length": 3.9974417686462402, "mode": "train", "epoch": 43}
{"loss": 0.557662844657898, "acc": 0.7119141221046448, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.5253768563270569, "acc": 0.7358378171920776, "length": 3.997528314590454, "mode": "train", "epoch": 44}
{"loss": 0.5550198554992676, "acc": 0.7175780534744263, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.5174137949943542, "acc": 0.7430336475372314, "length": 3.9969799518585205, "mode": "train", "epoch": 45}
{"loss": 0.5507140159606934, "acc": 0.7259765267372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.5189377069473267, "acc": 0.7425235509872437, "length": 3.9945614337921143, "mode": "train", "epoch": 46}
{"loss": 0.5534064769744873, "acc": 0.7222656607627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.5089204907417297, "acc": 0.7456042766571045, "length": 3.997910737991333, "mode": "train", "epoch": 47}
{"loss": 0.5519236922264099, "acc": 0.7263671159744263, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.503839910030365, "acc": 0.7488240599632263, "length": 3.99746036529541, "mode": "train", "epoch": 48}
{"loss": 0.5419617295265198, "acc": 0.7359375357627869, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.5025454163551331, "acc": 0.75035560131073, "length": 3.995638608932495, "mode": "train", "epoch": 49}
{"loss": 0.5390511751174927, "acc": 0.7332030534744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.4941159784793854, "acc": 0.7526162266731262, "length": 3.99774169921875, "mode": "train", "epoch": 50}
{"loss": 0.5351946353912354, "acc": 0.7378906607627869, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.4926227033138275, "acc": 0.7558764219284058, "length": 3.9992525577545166, "mode": "train", "epoch": 51}
{"loss": 0.5368869304656982, "acc": 0.7345702648162842, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.48644813895225525, "acc": 0.7561250925064087, "length": 3.9994349479675293, "mode": "train", "epoch": 52}
{"loss": 0.5384106040000916, "acc": 0.7376952767372131, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.4862809479236603, "acc": 0.7578521370887756, "length": 3.9986417293548584, "mode": "train", "epoch": 53}
{"loss": 0.5378414988517761, "acc": 0.7427734732627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.48454514145851135, "acc": 0.7586667537689209, "length": 3.997727632522583, "mode": "train", "epoch": 54}
{"loss": 0.5234485864639282, "acc": 0.7400389909744263, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.4804694354534149, "acc": 0.7615320682525635, "length": 3.9995005130767822, "mode": "train", "epoch": 55}
{"loss": 0.5245620012283325, "acc": 0.7388672232627869, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.4783727526664734, "acc": 0.7622512578964233, "length": 3.9995172023773193, "mode": "train", "epoch": 56}
{"loss": 0.523603081703186, "acc": 0.7439452409744263, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.47520607709884644, "acc": 0.7645311951637268, "length": 3.999513864517212, "mode": "train", "epoch": 57}
{"loss": 0.5180903673171997, "acc": 0.74609375, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.4742496609687805, "acc": 0.7668857574462891, "length": 3.9999403953552246, "mode": "train", "epoch": 58}
{"loss": 0.5282044410705566, "acc": 0.7437500357627869, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.4702408015727997, "acc": 0.7682716846466064, "length": 3.996574878692627, "mode": "train", "epoch": 59}
{"loss": 0.5241257548332214, "acc": 0.7447265982627869, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.46531471610069275, "acc": 0.7689840793609619, "length": 3.9997575283050537, "mode": "train", "epoch": 60}
{"loss": 0.5282868146896362, "acc": 0.7392578125, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7066665887832642
